gems/pending/spec/util/miq_process_spec.rb

```diff
@@ -0,0 +1,19 @@
+require 'util/miq-process'
+
+describe MiqProcess do
+  context ".command_line" do
+    it "exited process" do
+      allow(Sys::ProcTable).to receive(:ps).and_return nil
+      expect(described_class.command_line(123)).to eq ""
+    end
+
+    it "no permissions to proctable info" do
+      allow(Sys::ProcTable).to receive(:ps).and_return(double(:cmdline => nil))
+      expect(described_class.command_line(123)).to eq ""
+    end
+
+    it "normal case" do
+      expect(described_class.command_line(Process.pid).length).not_to eq 0
```

Comment:
Since the test is really checking against a String, I think it might be clearer if the RHS was a String check... i.e. `expect(described_class.command_line(Process.pid)).not_to be_empty`